test/xpu_api/containers/sequences/array/zero_sized_arrays_g.pass.cpp

```diff
@@ -46,7 +46,7 @@ int
 main()
 {
     auto ret = kernel_test();
-    EXPECT_TRUE(ret, "Wrong result of work with in emply dpl::array in kernel_test");
+    EXPECT_TRUE(ret, "Wrong result of work with in empty dpl::array in kernel_test");
```

Comment:
```suggestion
    EXPECT_TRUE(ret, "Wrong result of work with empty dpl::array in kernel_test");
```
Small grammatical error.